pkg/operator/staticpod/controllers.go

```diff
@@ -124,6 +131,11 @@ type Builder interface {
 	ToControllers() (manager.ControllerManager, error)
 }
 
+func (b *staticPodOperatorControllerBuilder) WithExtraNodeSelector(extraNodeSelector labels.Selector) Builder {
```

Comment:
What about other controllers potentially listing the master nodes? As I understand, we will only run the etcd member on the arbiter node. Is that correct?

Reply:
Right, exactly, so unless explicitly required the other controllers should run only on masters, but if they come to the conclusion they need to also run on the arbiter, we can have a path for them to do it